### PR TITLE
Setup landing page

### PR DESCRIPTION
**Pull request by hopeaisulu**

### CODE REVIEW

File: client/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
-    imports: [RouterTestingModule],
-    declarations: [AppComponent]
-  }));
+  beforeEach(() =>
+    TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule,
+        EmployeeAppModule,
+        MatToolbarModule,
+        MatIconModule,
```
Comment (networkflow) [inline client/src/app/app.component.spec.ts:16]: Nice job figuring out about the updated imports! Was it hard to debug this? (It took me probably 30-60 minutes to figure out what was going on with this.)

File: client/src/app/employee-app/employee-app-root/employee-app-root.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<mat-toolbar color="primary">
+  <span class="example-spacer">Choco-Pie Factory</span>
```
Comment (networkflow) [inline client/src/app/employee-app/employee-app-root/employee-app-root.component.html:2]: 🍫🥧🏭 = 😄

**Review by networkflow — APPROVED** [review]
Great work overall!

First commit:
* Looks perfect!

Second commit:
* The summary is probably a little more detailed than you need:
  * I know I talked a lot about including specific commands: thank you for doing that! However for things like `ng generate`, that's a very common Angular command, so it's unlikely that reviewers (or yourself in the future) would want to know that (unless there was something particularly complicated or interesting about it), so I'd just say something like "Created a new EmployeeApp module".
  * For things like updating module imports/exports, that's also usually not something you'd need to mention in the summary: it's an implementation detail, and reviewers can see it from the code.
  * "`Use toolbar and icon to build a landing page that says 'Hello Aisulu'`": I like this line in your summary, it's concise and describes what you did!

If it helps, here's my entire summary from the second commit:
```
Summary:
* Created an employee-app Angular module
* Added a basic toolbar and welcome message to the root component of this module
* Replaced the default Angular homepage with this component
```
I just focused on the high-level things I did (and in this case I didn't feel the commands I ran were important/interesting enough to include). This isn't the "right answer", but it's what I did :)

The test plan in your second commit is great: if you wanted to include a screenshot (you can upload them to the PR), it would probably save you the effort of writing so many words, but for something simple like this isn't not a big deal either way.

File: client/src/app/app.component.spec.ts (hunk 1 above)
Comment (hopeaisulu) [reply client/src/app/app.component.spec.ts:16]: Actually FuMing told me that I also needed to update in the tests.

**Pull request merged**